templates/project-ts/.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Because the provided code in the `templates` directory is TypeScript, I think we have to make sure eslint can parse TypeScript. Can you add this in? I think this can be done with the following NPM package: https://www.npmjs.com/package/@typescript-eslint/eslint-plugin